src/ORM/Marshaller.php

```diff
@@ -105,6 +105,18 @@ public function one(array $data, array $options = [])
         $propertyMap = $this->_buildPropertyMap($options);
 
         $schema = $this->_table->schema();
+        $primaryKey = $schema->primaryKey();
+
+        if ( array_intersect($primaryKey, array_keys($data)) == $primaryKey ) {
```

Comment:
The idea of this condition is exactly to check if the primary key is present. This condition is made to support a primary key wich is composed by 1 or more fields. I don't get exactly what you said about associated values, kindly ask you to explain a bit more what you mean. Thanks!